Makefile

```diff
@@ -3,34 +3,34 @@ SHELL = /bin/bash
 build:
 	docker build \
 		--build-arg GDAL_VERSION=3.0 \
-		--tag remotepixel/amazonlinux:gdal3.0-py3.7-cogeo .
+		--tag vexcel/amazonlinux:gdal3.0-py3.7-osgeo .
```

Comment:
Just a little rebranding here. 
This helps differentiate between the docker images we consume and the docker images we depend on.
For instance if you run `docker images` this helps us to clearly differentiate between our images and external images.